src/plugins/internalnotification/internalnotification.c

```diff
@@ -250,7 +316,7 @@ static int keySetContainsSameOrBelow (Key * check, KeySet * ks)
  *                  e.g. elektraInternalnotificationGet or elektraInternalnotificationCommit)
  *
  */
-void elektraInternalnotificationUpdateRegisteredKeys (Plugin * plugin, KeySet * keySet)
+/*void elektraInternalnotificationUpdateRegisteredKeys (Plugin * plugin, KeySet * keySet)
```

Comment:
Please remove code instead of commenting out, esp. before merging into master.

Reply:
done